app/cmd.go

```diff
@@ -707,6 +709,8 @@ func init() {
 	RootCmd.Flags().String("tls-cert-data", "", "Specifies a PEM-encoded certificate.  Has no effect if --listen-tls is unset.")
 	RootCmd.Flags().String("tls-key-data", "", "Specifies a PEM-encoded private key.  Has no effect if --listen-tls is unset.")
 
+	RootCmd.Flags().Duration("engine-cycle-time", def.EngineCycleTime, "Time between engine cycles.")
```

Comment:
Just curious - what is the use case for being able to specify the engine cycle time in this PR? Or is this just a nice-to-have for our tests going forward?

Reply:
In our other testing (e.g., Cypress), we've wired in manual hooks end-to-end to trigger engine cycles using a proxy, process wrapper, webhook, and signal passing; this achieves the same goal (makes tests run quick, instead of waiting 5sec) but without the complexity of implementing/copying all the required layers for the signal-passing approach.

We could always add that later, but this lets us start simple and clean without sacrificing run time.